src/Components/Components/src/ComponentActivator.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Note to self: rename file

Reply:
Sorry about that, I will rename the file now.

Reply:
No worries! I'm working on this PR now so if possible it would be good not to make further changes in case we conflict with each other. Is that OK?

Reply:
Already renamed a file, didn't think you would be that fast to start working :)
OK, I will stop with changes from now on so you can continue. Please let me know if you need anything else.

PS. I really appreciate all your work. Thank you!